lib/grammers-tl-gen/src/enums.rs

```diff
@@ -17,7 +17,7 @@ use std::collections::HashSet;
 use std::io::{self, Write};
 
 /// Types that implement Copy from builtin_type
-const COPY_TYPES: [&'static str; 5] = ["bool", "f64", "i32", "i64", "u32"];
+const COPY_TYPES: [&str; 7] = ["bool", "f64", "i32", "i64", "u32", "[u8; 16]", "[u8; 32]"];
```

Comment:
This is a behavior change. Do we need to do this in this PR?

Reply:
In generated.rs int128 and int256 used .clone() and this Is unecessary because they implement Copy

Reply:
Ah, so clippy was complaining about the generated code. That's fair.